salt/hg/init.sls

```diff
@@ -152,6 +153,10 @@ apache2:
       - file: /etc/apache2/sites-enabled/*
       - file: /etc/apache2/mods-enabled/*
       - file: /etc/ssl/private/hg.psf.io.pem
+  cmd.run:
```

Comment:
Still think we could use the apache module state instead of a command! https://docs.saltproject.io/en/latest/ref/states/all/salt.states.apache_module.html